tests/test_invoice.py

```diff
@@ -195,6 +206,57 @@ def make_assertion(*args, **_):
             protect_content=True,
         )
 
+    @pytest.mark.dev
```

Comment:
```suggestion
```

Reply:
it really keeps coming back

Reply:
Sometimes harshil makes me feel like I don't even read while reviewing 🙈 😄

Reply:
And me like I can't code :(

Reply:
> it really keeps coming back

we can like, have a git hook which removes that, but its a bit too overkill haha

Reply:
true, a bit overkill